translations/messages.fr.xlf

```diff
@@ -202,8 +206,8 @@
         <source>Put your hexagonal pieces next to another one, and create one of the 3 victorious shapes to win!</source>
         <target>Placez vos pièces hexagonales les unes à côté des autres et soyez le premier à créer une des trois formes requises pour gagner !</target>
       </trans-unit>
-      <trans-unit id="5829078581404563552" datatype="html">
-        <source>The Vikings checkers! Invaders must capture the King, defender must make him escape!</source>
+      <trans-unit id="5953008102885748258" datatype="html">
+        <source>The Vikings checkers! Invaders must capture the king, defender must make him escape!</source>
```

Comment:
tiens ça serait pas plutôt "The Viking checkers" ou "The Vikings' chekers" (les dames vikings (pas d'accord d'adjectif en anglais) ou les dames des vikings) ?

Reply:
t'as plussé mais pas changé :D

Reply:
Je pige pas ce que tu veux alors.
<trans-unit id="5953008102885748258" datatype="html">
        <source>The Vikings checkers! Invaders must capture the king, defender must make him escape!</source>
Localement, c'est pushé, et le check-translation.sh est très content.

Reply:
ben au choix:
  - The Viking checkers
  - The Vikings' checkers
Mais pas "The Vikings checkers" (incorrect langagièrement)